src/app/home/home.component.ts

```diff
@@ -48,41 +48,22 @@ export class HomeComponent implements OnInit {
     this.stats = this.route.snapshot.data.stats;
     this.highlightCards = this.route.snapshot.data.highlights;
     const queryParams = this.route.snapshot.queryParams;
-
-    if (environment.environmentId !== 'production') {
-      if (queryParams?.simulatedDate) {
```

Comment:
I think it makes sense to lose simulated dates now the logic is driven by factors we can edit in SF data at will.

Reply:
I think so, the simulated date was only used to control the redirect. If we want it it probably needs to be sent to SF as part of the query for highlighted campaigns so we can preview what that will return. Makes sense to take it away until and unless we need to do that.